### ISSUE TITLE
Console warning about aria-hidden on Cart/Checkout block coupon form.

### ISSUE THREAD

**Issue by opr** [woothemes/woocommerce#55039]
### Prerequisites

- [x] I have carried out troubleshooting steps and I believe I have found a bug.
- [x] I have searched for similar bugs in both open and closed issues and cannot find a duplicate.

### Describe the bug

Note: I could only reproduce this in Edge browser. It did not show on Firefox or Safari.

An error is shown in the JS console when applying a coupon:

```
Blocked aria-hidden on an element because its descendant retained focus. The focus must not be hidden from assistive technology users. Avoid using aria-hidden on a focused element or its ancestor. Consider using the inert attribute instead, which will also prevent focus. For more details, see the aria-hidden section of the WAI-ARIA specification at https://w3c.github.io/aria/#aria-hidden.
Element with focus: input
Ancestor with aria-hidden:

<div class="" aria-hidden="false"><div class="wc-block-components-totals-coupon__content"><form class="wc-block-components-totals-coupon__form" id="wc-block-components-totals-coupon__form"><div class="wc-block-components-text-input wc-block-components-totals-coupon__input is-active"><input type="text" id="wc-block-components-totals-coupon__input-coupon" autocapitalize="off" autocomplete="off" aria-label="Enter code" aria-invalid="false" title="" value="10off"><label for="wc-block-components-totals-coupon__input-coupon">Enter code</label></div><button type="submit" class="wc-block-components-button wp-element-button wc-block-components-totals-coupon__button contained" style=""><span class="wc-block-components-button__text">Apply</span></button></form></div></div>
```

### Expected behavior

The error should not be shown, the element should not become aria-hidden, or it should be unfocused.

### Actual behavior

The error is shown.

### Steps to reproduce

1. Add a coupon to your site.
2. Go to the Cart block and open the JS console.
3. Apply the coupon.
4. The error will be shown in the JS console.

### WordPress Environment

n/a

### Isolating the problem

- [x] I have deactivated other plugins and confirmed this bug occurs when only WooCommerce plugin is active.
- [x] This bug happens with a default WordPress theme active, or [Storefront](https://woocommerce.com/products/storefront/).
- [x] I can reproduce this bug consistently using the steps above.

**Comment by Kallyan01** [comment]
@opr I can't reproduce this issue on edge browser . Could you share more details like browser version , OS those might help . Thanks !

**Comment by opr** [comment]
Hi @Kallyan01 - I just tested and couldn't reproduce either, maybe a bug in edge that was fixed. Let's close the issue for now and if I spot it again I will reopen. Thanks for your help!
